site/en/docs/configurable-attributes.md

```diff
@@ -364,6 +364,11 @@ Platforms are still under development. See the
 
 `select` can appear multiple times in the same attribute:
 
+Note: Some restrictions apply on what can be combined in the select's values:
```

Comment:
Would this paragraph make more sense after the example? 

Also, as a grammar nit I'd omit "Note:" to get the same message through more concisely.

And one more grammar nit: add backtick around `select` so it appears code-style.

Reply:
I put it before the example that one would not easily overlook these restrictions.

I assumed that the `Note:` would yield something like this Note-box: https://bazel.build/reference/exec-groups#using-exec-groups-for-exec-properties

![image](https://user-images.githubusercontent.com/21087362/174898519-1b7180ce-40ba-4c61-a1b7-65c4817ccbc7.png)

Maybe if it's formatted like this, moving it after the example is the right choice. Then it would still catch the readers eye. wdyt?

Reply:
You're right, I'm sorry! I'd definitely support that formatting. Still after the example IMO.